helm/platform-health/templates/deployment.yaml

```
@@ -0,0 +1,34 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}--{{ .Chart.Name }}
+  {{- $deployment := .Values.deployment | default dict }}
+  {{- $labels := mergeOverwrite (.Values.common_labels | default dict) ($deployment.labels | default dict) | toYaml | nindent 4 }}
+  labels: {{ $labels }}
+  {{- $annotations := mergeOverwrite (.Values.common_annotations | default dict) ($deployment.annotations | default dict) | toYaml | nindent 4 }}
+  annotations: {{ $annotations }}
+spec:
+  replicas: 1
```

```suggestion
  replicas: {{ .Values.replicaCount }}
```
Would be better, I think? We should also allow specification of custom `nodeSelector`, `tolerations` and `topologySpreadConstraints` (all default empty, of course).

---

Of course!

---

It would be really nice to also have probing logic added to `platform-health`... I will take care of this.